src/index.css

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
Please see if this is required.

Reply:
removed as we are using electron app. this is useful for browser view